packages/datagateway-dataview/src/views/card/isis/isisInvestigationsCardView.component.tsx

```diff
@@ -229,7 +231,20 @@ const ISISInvestigationsCardView = (
         {
           icon: <Assessment />,
           label: t('investigations.instrument'),
-          dataKey: 'investigationInstruments[0].instrument.fullName',
+          dataKey: 'investigationInstruments.instrument.fullName',
+          // eslint-disable-next-line react/display-name
+          content: (investigation: Investigation) => {
+            const instrument = nestedValue(
+              investigation,
+              'investigationInstruments[0].instrument.fullName'
+            );
+            return (
+              <ArrowTooltip title={instrument}>
+                <Typography>{instrument}</Typography>
+              </ArrowTooltip>
+            );
```

Comment:
So the error avoided by this eslint-ignore line is caused by returning JSX (or essentially a component) that is missing a `displayName`. This can be fixed by wrapping the return statement in a function named with PascalCase style. The function name is adopted as the 'component' display name. For example:

```
const instrument = nestedValue(
    investigation,
    'investigationInstruments[0].instrument.fullName'
);
return function MyComponent(): React.ReactNode {
    return (
        <ArrowTooltip title={instrument}>
            <Typography>{instrument}</Typography>
        </ArrowTooltip>
    );
};
```
So with this, you should be able to remove that eslint-ignore line :)

Reply:
This is really helpful @sam-glendenning, I had been wondering how these errors could be have been fixed!